### DISCUSSION TITLE
jquery.liveaddress - smarty api

### DISCUSSION THREAD

**Discussion by santoshgitrepo** [community/community#140289] [Programming Help]
### Select Topic Area

Question

### Body

is there any way we can remove the default  pop up validation message and show this message in label. PFA, image for details
<img width="513" alt="img-1" src="https://github.com/user-attachments/assets/3843a99f-2e85-4307-bfd1-b34c7b7b1fd2">

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @santoshgitrepo ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
